### PR TITLE
#16 adding list view PR

### PR DESCRIPTION
**Pull request by MaximeJoanisse**
Unit Testing was not done in this PR, My fault due to previous parts having more difficulty than anticipated:

What was done:

- Dynamic pages for table name
- Sidebar of table names
- Connect VMD with tableName
- Added a page for Enum view

### CODE REVIEW

File: UInnovateApp/src/App.css [hunk 1]
```diff
@@ -40,3 +40,6 @@
 .read-the-docs {
   color: #888;
 }
+.sidebar{
```
Comment (lyubageorgieva) [inline UInnovateApp/src/App.css:43]: This is empty, what do we need it for?

**Review by lyubageorgieva — COMMENTED** [review]
Everything seems to be running well, it's just I'm wondering why we have an unused sidebar style in App.css

File: UInnovateApp/src/App.css (hunk 1 above)
Comment (MaximeJoanisse) [reply UInnovateApp/src/App.css]: its been removed thank you for the catch.

**Review by lyubageorgieva — APPROVED** [review]
Looks good!

**Review by SamRfk — APPROVED** [review]
Everything looks good, I'll probably be changing a few things when implementing the enum view as well - and thank you for making the routes already for it!

**Pull request merged**